samtranslator/plugins/application/serverless_app_plugin.py

```diff
@@ -187,7 +190,8 @@ def on_before_transform_resource(self, logical_id, resource_type, resource_prope
         self._check_for_dictionary_key(logical_id, resource_properties, [self.LOCATION_KEY])
 
         # If location isn't a dictionary, don't modify the resource.
-        if not isinstance(resource_properties[self.LOCATION_KEY], dict):
+        if (not isinstance(resource_properties[self.LOCATION_KEY], dict)
```

Comment:
Can you make it a util function to check these conditions? You have the same check in the function above.

Reply:
Good idea